### DISCUSSION TITLE
Errors in loop

### DISCUSSION THREAD

**Discussion by Iqra928** [AammarTufail/python_ka_chilla2023#115] [Q&A]
Sir I have an error in loops. due to this error next chapter is also corrupted. Loop section shows the output of infinit loop. I deleted the file, and retried but the error persistes. kindly help how to remove the error

**Comment by Hurairah17070** [comment]
i think you should identify the problem that is creating mess.. fix it and you're good to go.

**Comment by kousarraza** [comment]
Which error please mentioned it send SS?

**Comment by Hurairah17070** [comment]
You're not ammar?

**Comment by marshadanwer** [comment]
Yes I'm not Ammar!
